src/css/flags/ads.less

```diff
@@ -78,7 +78,7 @@
 
 }
 
-.jwplayer.jw-flag-ads-googleima {
+.jwplayer.jw-flag-ads.jw-flag-ads-googleima {
```

Comment:
This results in controls being shown all the time for googima ads instead of when `vpaidcontrols === true`.